std_msgs/String name
float32[] position
float32[] orientation
---
bool success